src/users.ts

```diff
@@ -1,36 +1,79 @@
 import express, { Request, Response } from "express";
-import jwtDecode from "jwt-decode";
+import { createSolidTokenVerifier, RequestMethod } from '@solid/access-token-verifier';
 import { INTEROP } from "@janeirodigital/interop-namespaces";
+import SessionStorage from "./session-storage";
 
 const usersRouter = express.Router({ caseSensitive: false });
 
-usersRouter.get("/:id", async (req: Request, res: Response) => {
-  const tokenHeader = req.header("Authorization")?.split(" ")[1]; // Leave the `DPoP ` behind.
+usersRouter.get("/:webIdB64", async (req: Request, res: Response) => {
 
-  if (!tokenHeader) {
+  const webIdFromAgent = Buffer.from(req.params.webIdB64, "base64").toString("utf-8")
+  console.log(webIdFromAgent)
+  const sai = SessionStorage.get(webIdFromAgent);
+
+  // TODO rethink how SessionStorage gets pupulated from redis - most likely on start
```

Comment:
Currently, we set SAI sessions in [`buildSaiSession`](https://github.com/janeirodigital/sai-impl-service/pull/7/files#diff-96998a2148aa3d52ad3e80303d0ad88fc43a4afd08966b5330a5657021ae52bbR13) I think they need to be populated from oidcSessions in Redis when the service process starts.